DocumentService/Controllers/DocumentsController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace DocumentService.Controllers
+{
+    [Route("api/")]
+    [ApiController]
+    public class DocumentsController : ControllerBase
+    {
+
+        public DocumentsController()
+        {
+
+        }
+
+        /// <summary>
+        /// Upload a document.
+        /// </summary>
+        /// <param name="CorrelationId">Correlation identifier of the operation.</param>
+        /// <param name="UserName">Azure AD identifier of the user uploading the document.</param>
+        /// <param name="FileName">File name of the document.</param>
+        /// <param name="ShortDescription">Short description of the document.</param>
+        /// <param name="SubmissionMethod">Indicates how the file was submitted to Transport Canada. (“FAX”, “MAIL”, “EMAIL”).</param>
+        /// <param name="FileLanguage">Language of document being uploaded. (1) English, (2) French.</param>
+        /// <param name="FileType">Type of file being uploaded. (“PDF”) pdf file, (“IMG”) Image file, (“TXT”) text file.</param>
+        /// <param name="DocumentTypes">Document type of the uploaded document. A document can have multiple types associated with it. The document type id is supplied by the client using the document service.</param>
+        /// <param name="Bytes">Document being uploaded.</param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("v1/documents")]
+        [ProducesResponseType(StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult UploadDocument(int CorrelationId, string UserName, string FileName, string? ShortDescription, string? SubmissionMethod, int? FileLanguage, string? FileType, List<string>? DocumentTypes, string Bytes)
+        {
+
+            return Ok();
+        }
+
+        /// <summary>
+        /// Retrieve all metadata for all specified documents. 
+        /// </summary>
+        /// <param name="CorrelationId">Correlation identifier of the operation.</param>
+        /// <param name="ListOfIds">List of identifiers of the uploaded documents. Should be like 1,2,3,4</param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("v1/documents")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult GetAllSpecifiedDocuments(int CorrelationId, string ListOfIds)
```

Comment:
Same thing here we can remove the correlation Id. The ListOfIds should be a list of strings. Unless there's a reason I don't know about to have it as a string.